src/resolvers/subscription/metrics/queries.js

```diff
@@ -0,0 +1,146 @@
+const components = `scheduler|webserver|worker|flower|redis|pgbouncer|statsd`;
+const namespace = `.*-.*-.*-[0-9]{4}`;
+
+// Poll interval (milliseconds)
+const step = config.get("prometheus.pollInterval");
+
+export default function queries(deployment) {
+  // UNIX start timestamp, minus step
+  const start = Math.round((new Date().getTime() - step) / 1000);
+
+  // UNIX current time timestamp
+  const now = Math.round(new Date().getTime() / 1000);
+
+  // Query builders (Prom uses different endpoints for different metrics)
+  const query = ql => `query?query=${encodeURI(ql)}&time=${now}`;
+  const rangeQuery = ql =>
+    `query_range?start=${start}&end=${now}&step=${step}&query=${encodeURI(ql)}`;
+
+  return [
+    {
+      name: "deploymentStatus",
+      query: query(`sum by (release) (
+          kube_pod_container_status_running{namespace=~"${namespace}", release=~"${deployment}"}
+        ) -
+        count by (release) (
+          kube_pod_container_status_running{namespace=~"${namespace}", release=~"${deployment}"}
+        )`)
+    },
+    {
+      name: "schedulerHeartbeat",
+      query: query(
+        `round(rate(airflow_scheduler_heartbeat{deployment=~"${deployment}", type="counter"}[1m]) * 5)`
+      )
+    },
+    {
+      name: "maxPods",
+      query: rangeQuery(
+        `sum(kube_resourcequota{resource="pods", type="hard", namespace=~"${namespace}", release=~"${deployment}"})`
+      )
+    },
+    {
+      name: "cpuMax",
+      query: rangeQuery(
+        `sum(kube_resourcequota{resource="limits.cpu", type="hard", namespace=~"${namespace}", release=~"${deployment}"})`
+      )
+    },
+    {
+      name: "memoryMax",
+      query: rangeQuery(
+        `sum(kube_resourcequota{resource="limits.memory", type="hard", namespace=~"${namespace}", release=~"${deployment}"})`
+      )
+    },
+    {
+      name: "heartbeat",
+      query: query(
+        `sum(airflow_scheduler_heartbeat{deployment=~"${deployment}"})`
+      )
+    },
+    {
+      name: "dagCount",
+      query: query(`sum(airflow_dagbag_size{deployment=~"${deployment}"})`)
+    },
+    {
+      name: "zombieCount",
+      query: query(`sum(airflow_zombies_killed{deployment=~"${deployment}"})`)
+    },
+    {
+      name: "taskSuccesCount",
+      query: query(`sum(airflow_ti_successes{deployment=~"${deployment}"})`)
+    },
+    {
+      name: "taskSuccessRate",
+      query: query(
+        `rate(airflow_operator_successes{deployment=~"${deployment}"}[3m])`
+      )
+    },
+    {
+      name: "taskFailRate",
+      query: query(
+        `rate(airflow_operator_failures{deployment=~"${deployment}"}[3m])`
+      )
+    },
+    {
+      name: "taskFailCount",
+      query: query(`sum(airflow_ti_failures{deployment=~"${deployment}"})`)
+    },
+    {
+      name: "connectCount",
+      query: query(
+        `sum(pgbouncer_exporter_database_current_connections{name!="pgbouncer", deployment=~"${deployment}"})`
+      )
+    },
+    {
+      name: "waitingCount",
```

Comment:
```suggestion
      name: "dbConnwaitingCount",
```

Not suer where these names are used, but might be a bit ambigious otherwise? Similar for `connectCount`

Reply:
This makes sense.